src/Resources/views/tailwind_2_layout.html.twig

```diff
@@ -11,7 +11,7 @@
                 }))
             }) }}
         >
-            <turbo-frame id="whatwedo_table_{{ table.identifier }}" data-turbo="true" target="_top" {{ stimulus_action('@araise/table-bundle/table_select', 'tableTargetChanged', 'turbo:frame-load') }}>
+            <turbo-frame id="whatwedo_table_{{ table.identifier }}" data-turbo="{{ araise_turbo_enabled ? 'true' : 'false' }}" target="_top" {{ stimulus_action('@araise/table-bundle/table_select', 'tableTargetChanged', 'turbo:frame-load') }}>
```

Comment:
create another twig extension for the parameter `araise_table.enable_turbo` and use this one.